lib/encoding/zstd/zstd.go

```diff
@@ -0,0 +1,19 @@
+// +build cgo
```

Comment:
Rename this file to `zstd_cgo.go`, so its name is more consistent with `zstd_pure.go`